pipeline/frontend/yaml/compiler/convert.go

```diff
@@ -84,7 +84,7 @@ func (c *Compiler) createProcess(name string, container *yaml.Container, section
 		}
 	}
 
-	if matchImage(container.Image, c.escalated...) {
+	if matchImage(container.Image, c.escalated...) && container.IsPlugin() {
```

Comment:
@qwerty287 Do we need to change line 23: `privileged  = container.Privileged` to false?